model/orion/dffml_model_orion/orion_model.py

```diff
@@ -0,0 +1,231 @@
+    )
+    predict: str = field(
+        "The start and end values of your ground truth as a csv file",
+        default="",
+    )
+    accuracy: str = field("The ground truth ", default="")
+    hyperparameters: tuple = field(
+        "Hyperparameters for training the model",
+        default=("keras.Sequential.LSTMTimeSeriesRegressor#1", {"epochs": 5}),
+    )
+    pipeline: str = field(
+        "Orion pipeline to be used in model training",
+        default="lstm_dynamic_threshold",
+    )
+    directory: pathlib.Path = field(
+        "Filepath to save model configs", default="model"
+    )
+
+
+@entrypoint("orion")
+class OrionModel(SimpleModel):
+    r"""
+    Time Series anomaly detection using Orion.
+
+    The dataset used for these examples is generated through orion/tests/gen_data.py
+    Run it with python ./tests/gen_data.py
+
+    The same is also available in orion/examples/gen_data.py
+
+    **train.csv**
+
+    .. code-block::
+        :test:
+        :filepath: train.csv
+
+        File is generated through tests/gen_data.py
+
+    The dataset used for testing
+
+    **test.csv**
+
+    .. code-block::
+        :test:
+        :filepath: test.csv
+
+        File is generated through tests/gen_data.py
+
+    Train the model
+
+    .. code-block:: console
+        :test:
+
+        $ dffml train \
+            -model orion \
+            -model-data './tests/train.csv' \
+            -model-predict './tests/predict.csv' \
+            -model-accuracy './tests/test.csv' \
+            -model-hyperparameters "keras.Sequential.LSTMTimeSeriesRegressor#1"\,{\"epochs\": 5} \
+            -sources f=csv
+            -source-filename ./tests/train.csv
+
+    Assess the accuracy
+
+    .. code-block:: console
+        :test:
+
+        $ dffml accuracy \
+            -model orion \
+            --model-data './tests/train.csv' \
+            -model-predict './tests/predict.csv' \
+            -model-accuracy './tests/test.csv' \
+            -model-hyperparameters "keras.Sequential.LSTMTimeSeriesRegressor#1"\,{\"epochs\": 5} \
+            -sources f=csv
+            -source-filename ./tests/test.csv
+        accuracy     0.950341
+        f1           0.313208
+        recall       0.185682
+        precision    1.000000
+        dtype: float64
+        0.9503410641200546
+
+    Make a prediction
+
+    **predict.csv**
+
+    .. code-block::
+        :test:
+        :filepath: predict.csv
+
+        File is generated through tests/gen_data.py
+
+    .. code-block:: console
+        :test:
+
+        $ dffml predict all \                                                                     (gsoc)
+                 -model orion \
+                 -model-data './tests/train.csv' \
+                 -model-predict './tests/predict.csv' \
+                 -model-accuracy './tests/test.csv' \
+                 -model-hyperparameters "keras.Sequential.LSTMTimeSeriesRegressor#1"\,{\"epochs\": 5} \
```

Comment:
We'll leave hyperparameters like this for now.

- https://github.com/MLBazaar/MLPrimitives
- https://mlbazaar.github.io/MLBlocks/advanced_usage/hyperparameters.html
- https://sintel.dev/Orion/getting_started/quickstart.html